app/Resources/views/editCounter/general_stats.html.twig

```diff
@@ -126,7 +126,12 @@
                 <td>{{ ec.averageRevisionsPerPage }}</td>
             </tr>
             <tr>
-                <td>{{ msg('pages-created') }}</td>
+                <td>
+                    <a class="glyphicon glyphicon-info-sign" target="_blank" title="{{ msg('pages-created-tooltip') }}"
+                       href="https://xtools.readthedocs.io/en/latest/tools/editcounter.html#general-statistics">
+                    </a>
+                    {{ msg('pages-created') }}
+                </td>
```

Comment:
I've switched to the `tooltipcss` system, and added a couple of bits to that system; see what you think.